core/templates/dev/head/pages/exploration_player/PlayerTranscriptServiceSpec.js

```diff
@@ -86,7 +86,12 @@ describe('Player transcript service', function() {
       a: 'b'
     }, 'Content HTML', '<oppia-text-input-html></oppia-text-input-html>');
     lastAnswer = pts.getLastAnswerOnActiveCard(0);
-    expect(lastAnswer).not.toEqual(null);
+    var firstCard = pts.getCard(0);
```

Comment:
I don't understand why you're doing an expectation on inputResponsePairs here. Isn't this supposed to test lastAnswer instead?

Reply:
lastAnswer equals last entry in inputResponse from getLastAnswerOnActiveCard() if it is not last card.

Reply:
Sure, but this test method is supposed to test lastAnswer. Why not do `expect(lastAnswer).toEqual('first answer')` or similar?

Reply:
Done